File: Travel.Data/Travel.cs
using System;
using System.Collections.Generic;
using Travel.Common.Enums;

namespace Travel.Data
{
	public class Travel
	{
		public Travel()
		{
			MapPoints = new List<MapPoint>();
			Travelers = new List<Traveler>();
		}
		public int Id { get; set; }
		public int DriverId { get; set; }
		public int? Capacity { get; set; }
		public ICollection<Traveler> Travelers { get; set; }
		public ICollection<MapPoint> MapPoints { get; set; }
		public DateTime Startdate { get; set; }
		public DateTime Enddate { get; set; }
		public string Startpoint { get; set; }
		public string Endpoint { get; set; }
		public string DisplayName { get; set; }
		public string Mapsnapshot { get; set; }
		public LuggageType Luggage { get; set; }
	}
}

File: Travel.Web/Scripts/Application/Models/Travel.ts
/// <reference path="../_references.ts" />
module Application
{
    export class Travel {
        Capacity: number;
        Id: number;
        DriverId: number;
        Startdate: Date;
        Enddate: Date;
        MapPoints: Array<MapPoint>;
        Startpoint: string;
        Endpoint: string;
        Travelers: Array<Traveler>;
        DisplayName: string;
        Mapsnapshot: string;
        Friends: Array<MyFriend>;
        Luggage: LuggageType;
    }
} 